app/(dashboard)/settings/knowledge/suggestedKnowledgeBankItem.tsx

```
@@ -27,19 +27,20 @@ const SuggestedKnowledgeBankItem = ({ faq }: SuggestedKnowledgeBankItemProps) =>
   });
 
   return (
-    <div className="flex flex-col gap-2 border border-bright rounded-lg p-4">
+    <div className="flex flex-col gap-2 border border-bright rounded-lg p-4" data-testid="suggested-knowledge-item">
       <div className="flex-1 w-full text-left text-sm">
-        <Textarea value={content} onChange={(e) => setContent(e.target.value)} rows={5} />
+        <Textarea value={content} onChange={(e) => setContent(e.target.value)} rows={5} data-testid="suggested-knowledge-textarea" />
```

ESLint

[error] Replace `·value={content}·onChange={(e)·=>·setContent(e.target.value)}·rows={5}·data-testid="suggested-knowledge-textarea"` with `⏎··········value={content}⏎··········onChange={(e)·=>·setContent(e.target.value)}⏎··········rows={5}⏎··········data-testid="suggested-knowledge-textarea"⏎·······`

(prettier/prettier)